perfkitbenchmarker/managed_relational_db.py

```diff
@@ -0,0 +1,107 @@
+from abc import ABCMeta, abstractmethod
+import uuid
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+
+# TODO (ferneyhough): change to enum
+flags.DEFINE_string('database', None,
+                    'Managed database flavor to use (mysql, postgres)')
+flags.DEFINE_string('database_name', None,
```

Comment:
That is up to the implementation of the benchmark in PKB. If the sysbench benchmark requires a certain name, then that will be reflected in the spec.